crocks/List.spec.js

```diff
@@ -123,7 +123,7 @@ test('List value', t => {
 test('List toArray', t => {
   const a = List([ 'some-thing', ['else', 43], 34 ]).toArray()
 
-  t.same(a, [ 'some-thing', 'else', 43, 34 ], 'folds the wrapped array')
+  t.same(a, [ 'some-thing', ['else', 43], 34 ], 'provides the wrapped array')
```

Comment:
It is a bad test because it was wrong. It was not testing the right thing. Also because you did not take my suggestion it is now a brittle test